includes/widgets/text-editor.php

```diff
@@ -169,6 +169,9 @@ protected function register_controls() {
 				'selectors' => [
 					'{{WRAPPER}}' => 'column-gap: {{SIZE}}{{UNIT}};',
 				],
+				'condition' => [
+					'text_columns!' => [ '', '1' ],
+				],
```

Comment:
I'm removing this. Please create a separate PR for each issue. This PR adds responsive size to testimonial images.

```suggestion
```